wonderous-book/lib.typ

```diff
@@ -64,20 +64,14 @@
 
   // Configure page properties.
   set page(
-    numbering: "1",
-
     // The header always contains the book title on odd pages and
-    // the chapter title on even pages, unless the page is one
-    // that starts a chapter (the chapter title is obvious then).
+    // the author on even pages, unless the page is one that
+    // starts a chapter (the chapter title is obvious then).
+    // These values can easily be change below.
     header: context {
-      // Are we on an odd page?
-      if calc.odd(counter(page).get().first()) {
```

Comment:
By not having this check before the `all.any(..)` the page next to the chapter start is also affected. Is this intentional.

Reply:
The page next to the chapter start is effected anyway. Should this possibly be `calc.even`? The idea being that the chapter title page doesn't include the header, but the page following it does?

Reply:
I decided to add the header to the page after the chapter title since it's another page spread. I also added some code to skip headers & footers on empty pages (through a workaround in https://github.com/typst/typst/issues/2722).